components/meter/README.md

```diff
@@ -88,3 +88,11 @@ Then add the `d2l-meter-circle`, provide values for the properties, `value` and
 * `text` (optional): Context information about what the meter is about. This is for screen readers.
 	- `{%}` in the string will be replaced with percentage value.
 	- `{x/y}` in the string will be replaced with fraction with the proper language support.
+
+##A11Y##
```

Comment:
I think I'd call this section "Light Foreground", since technically it's not really an accessibility thing, it's more of a stylistic thing.